### PR TITLE
Feature: Disable all future questions + refactored EditButton.tsx

### PR DESCRIPTION
**Pull request by robinpiets**
Disable all future question if the current question is last of the stack. We need this because it is very hard to detect future open questions and this is causing bugs. We can fix this by stop using the combo of checker.stack and checker._getUpcomingQuestions(). We already have a ticket for that.

## Dear reviewer: Please test carefully with the dakraam and zonnepanelen checker with multiple addresses and make sure you try to break the checker by changing future questions

### CODE REVIEW

File: apps/client/src/components/Questions.js [hunk 1]
```diff
@@ -280,7 +292,10 @@ const Questions = ({
             key={`question-${q.id}-${index}`}
           >
             <QuestionAnswer
-              disabled={!!checker.isConclusive()}
+              disabled={
```
Comment (svenjens) [inline apps/client/src/components/Questions.js]: Maybe this is a bit hard to read. 
Can we make this a `const disabledFutureQuestions = !!checker.isConclusive() | |  (!checker.isConclusive() && disableFutureQuestions)`

File: apps/client/src/components/Questions.js [hunk 2]
```diff
@@ -203,6 +208,13 @@ const Questions = ({
         // Hide unanswered questions (eg: on browser refresh)
         if (!isCurrentQuestion && !userAnswer) return null;
 
+        // Disable all future question if this question is last of the stack
+        // We need this because it is very hard to detect future open questions and this is causing bugs
+        // @TODO: fix this by stop using the combo of checker.stack and checker._getUpcomingQuestions()
+        if (isCurrentQuestion && checker.stack.length === i + 1) {
+          disableFutureQuestions = true;
```
Comment (svenjens) [inline apps/client/src/components/Questions.js:215]: Should this be a state hook? It should rerender the component right?

File: apps/client/src/components/Questions.js [hunk 3]
```diff
@@ -280,7 +292,10 @@ const Questions = ({
             key={`question-${q.id}-${index}`}
           >
             <QuestionAnswer
-              disabled={!!checker.isConclusive()}
+              disabled={
+                !!checker.isConclusive() ||
```
Comment (afjlambert) [inline apps/client/src/components/Questions.js]: It returns a bool, no need to do double negation.

**Review by afjlambert — CHANGES_REQUESTED** [review]

File: apps/client/src/components/Questions.js (hunk 2 above)
Comment (robinpiets) [reply apps/client/src/components/Questions.js:215]: As explained this morning. Using a state hook will make this much more complicated

File: apps/client/src/components/Questions.js (hunk 1 above)
Comment (robinpiets) [reply apps/client/src/components/Questions.js]: Done!

File: apps/client/src/components/Questions.js (hunk 3 above)
Comment (robinpiets) [reply apps/client/src/components/Questions.js]: Good point!

**Review by svenjens — APPROVED** [review]

**Review by afjlambert — APPROVED** [review]

**Pull request merged**